### PR TITLE
Feat: modify style of issue kanban

### PR DESCRIPTION
**Pull request by sherotree**
## What this PR does / why we need it:
1. modify the style of issue kanban
2. add  auxiliary colors and default color

image

image

image

## I have checked the following points:
- [x] I18n is finished and updated by cli
- [x] Form fields validation is added and length is limited
- [x] Display normally on small screen
- [x] Display normally when some data is empty or null
- [x] Display normally in english mode

## Does this PR introduce a user interface change?

✅ Yes(screenshot is required)

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/common/components/tags/index.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import React from 'react';
+import { Tooltip, Popconfirm } from 'antd';
+import { Ellipsis, ErdaIcon } from 'common';
+import i18n from 'i18n';
+import './index.scss';
+
+interface ILabel {
```
Comment (McDaddy) [inline shell/app/common/components/tags/index.tsx:20]: may remove `I` prefix

File: shell/app/common/components/tags/index.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+export interface IProps extends Omit<IItemProps, 'label'> {
+  labels: ILabel[] | ILabel;
+  showCount?: number;
+  containerClassName?: string;
+}
+
+interface IItemProps {
+  label: ILabel;
+  maxWidth?: number;
+  colorMap?: Obj;
```
Comment (McDaddy) [inline shell/app/common/components/tags/index.tsx:35]: I guess k-v are both string, better be `Obj<string>`

File: shell/app/common/components/tags/index.tsx [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  checked?: boolean;
+  deleteConfirm?: boolean;
+  onDelete?: (p: ILabel) => void;
+}
+
+export const TagItem = (props: IItemProps) => {
+  const { label: _label, size = 'default', maxWidth, onDelete, deleteConfirm = true, colorMap, checked } = props;
+  const { label, color = 'blue' } = _label;
+  const [isChecked, setIsChecked] = React.useState(checked);
+  // compatible with gray which color is removed now
```
Comment (McDaddy) [inline shell/app/common/components/tags/index.tsx:46]: add `//TODO` mark

File: shell/app/common/components/tags/index.tsx [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+          zIndex={2010} // popconfirm zIndex is bigger than tooltip
+        />
+        {onDelete ? (
+          deleteConfirm ? (
+            <Popconfirm
+              title={`${i18n.t('common:confirm deletion')}?`}
+              arrowPointAtCenter
+              zIndex={2000} //  popconfirm default zIndex=1030, is smaller than tooltip zIndex=1070
+              onConfirm={(e) => {
+                e && e.stopPropagation();
```
Comment (McDaddy) [inline shell/app/common/components/tags/index.tsx:70]: this `e` is optional? if yes then `e?. stopPropagation()`

File: shell/app/common/components/tags/index.tsx [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+                onDelete(_label);
+              }}
+              onCancel={(e) => e && e.stopPropagation()}
+            >
+              <ErdaIcon size="16" className="cursor-pointer text-default-2 ml-0.5" type="close" color="currentColor" />
```
Comment (McDaddy) [inline shell/app/common/components/tags/index.tsx:75]: currentColor is the default value now, could remove it

File: shell/app/common/constants.ts [hunk 6]
```diff
@@ -51,3 +51,66 @@ export const DOC_CMP_CLUSTER_MANAGE = `${DOC_PREFIX}/manual/cmp/guide/cluster/ma
 // aliyun
 export const ALIYUN_APPLICATION =
   'https://account.aliyun.com/login/login.htm?oauth_callback=https%3A%2F%2Fdysms.console.aliyun.com%2Fdysms.htm%23%2Fdomestic%2Ftext%2Ftemplate%2Fadd';
+
+export const auxiliaryColorMap = {
```
Comment (McDaddy) [inline shell/app/common/constants.ts:55]: can these definition replace by the definition in theme-color?

File: shell/app/modules/project/pages/settings/components/project-label.scss [hunk 7]
```diff
@@ -29,7 +28,6 @@
     &::before {
       position: absolute;
       left: 4px;
-      content: '·';
```
Comment (McDaddy) [inline shell/app/modules/project/pages/settings/components/project-label.scss:32]: if remove content, is this `::before` still required?

File: shell/app/common/components/tags/index.tsx [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+          />
+        )}
+      </div>
+    </span>
+  );
+};
+
+const Tags = ({
+  labels: propsLabels,
+  showCount = 2,
```
Comment (McDaddy) [inline shell/app/common/components/tags/index.tsx:102]: I think `showCount` is very hard to guess the meaning. sounds like a boolean whether should show count number

better be `maxShowCount` if I understand correctly

File: shell/app/common/constants.ts (hunk 6 above)
Comment (sherotree) [reply shell/app/common/constants.ts:55]: 😂 Can't find a good solution for the time being

Comment (sherotree) [reply shell/app/common/constants.ts:55]: 😂 Can't find a good solution for the time being

**Review by McDaddy — APPROVED** [review]

**Pull request merged**